hydrus/data/db_models.py

```diff
@@ -8,9 +8,12 @@
 from typing import Any
 import datetime
 import uuid
+import os
 # from hydrus.settings import DB_URL
 
-engine = create_engine('sqlite:///database.db')
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.realpath(__file__)))) #to put database file in the rootdirectory
```

Comment:
This is where all issues occur. Our PEP8 test fails at 101 chars per line. Place the comment on top of the code.

Reply:
done, thanks for pointing out